### DISCUSSION TITLE
Obtain the current conversation ID in a custom tool

### DISCUSSION THREAD

**Discussion by hllshiro** [langgenius/dify#7884] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

Hello,

I would like to know if there is a way to obtain the current conversation_id in a custom tool. I have a requirement to distinguish between different conversations using the conversation_id and to temporarily store some data.

**Comment by hugoblanc** [comment]
I would love to have the answer of that question
